examples/local_eventstore/books/main.go

```diff
@@ -21,10 +24,10 @@ type bookStats struct {
 }
 
 func main() {
-	s, clean := createMemDB()
+	d, clean := createMemDB()
```

Comment:
I tried to find all these and rename them, but obv missed some. Thanks for cleaning up!

Reply:
Yep yep!